woocommerce-gateway-stripe.php

```diff
@@ -188,83 +188,84 @@ public function __construct() {
+				require_once __DIR__ . '/includes/payment-methods/class-wc-stripe-upe-payment-method-sofort.php';
+				require_once __DIR__ . '/includes/payment-methods/class-wc-stripe-upe-payment-method-multibanco.php';
+				require_once __DIR__ . '/includes/payment-methods/class-wc-stripe-upe-payment-method-link.php';
+				require_once __DIR__ . '/includes/payment-methods/class-wc-stripe-upe-payment-method-cash-app-pay.php';
+				require_once __DIR__ . '/includes/payment-methods/class-wc-stripe-upe-payment-method-wechat-pay.php';
+				require_once __DIR__ . '/includes/payment-methods/class-wc-gateway-stripe-bancontact.php';
+				require_once __DIR__ . '/includes/payment-methods/class-wc-gateway-stripe-sofort.php';
+				require_once __DIR__ . '/includes/payment-methods/class-wc-gateway-stripe-giropay.php';
+				require_once __DIR__ . '/includes/payment-methods/class-wc-gateway-stripe-eps.php';
+				require_once __DIR__ . '/includes/payment-methods/class-wc-gateway-stripe-ideal.php';
+				require_once __DIR__ . '/includes/payment-methods/class-wc-gateway-stripe-p24.php';
+				require_once __DIR__ . '/includes/payment-methods/class-wc-gateway-stripe-alipay.php';
+				require_once __DIR__ . '/includes/payment-methods/class-wc-gateway-stripe-sepa.php';
+				require_once __DIR__ . '/includes/payment-methods/class-wc-gateway-stripe-multibanco.php';
+				require_once __DIR__ . '/includes/payment-methods/class-wc-gateway-stripe-boleto.php';
+				require_once __DIR__ . '/includes/payment-methods/class-wc-gateway-stripe-oxxo.php';
+				require_once __DIR__ . '/includes/payment-methods/class-wc-stripe-payment-request.php';
+				require_once __DIR__ . '/includes/payment-methods/class-wc-stripe-express-checkout-element.php';
+				require_once __DIR__ . '/includes/payment-methods/class-wc-stripe-express-checkout-helper.php';
+				require_once __DIR__ . '/includes/payment-methods/class-wc-stripe-express-checkout-ajax-handler.php';
+				require_once __DIR__ . '/includes/compat/class-wc-stripe-woo-compat-utils.php';
+				require_once __DIR__ . '/includes/connect/class-wc-stripe-connect.php';
+				require_once __DIR__ . '/includes/connect/class-wc-stripe-connect-api.php';
+				require_once __DIR__ . '/includes/class-wc-stripe-order-handler.php';
+				require_once __DIR__ . '/includes/payment-tokens/class-wc-stripe-payment-tokens.php';
+				require_once __DIR__ . '/includes/class-wc-stripe-customer.php';
+				require_once __DIR__ . '/includes/class-wc-stripe-intent-controller.php';
+				require_once __DIR__ . '/includes/admin/class-wc-stripe-inbox-notes.php';
+				require_once __DIR__ . '/includes/admin/class-wc-stripe-upe-compatibility-controller.php';
+				require_once __DIR__ . '/includes/migrations/class-allowed-payment-request-button-types-update.php';
+				require_once __DIR__ . '/includes/class-wc-stripe-account.php';
+				
 				new Allowed_Payment_Request_Button_Types_Update();
```

Comment:
There were some file path changes that required a rebase here; I used diff to make sure the names match.